cypress-io/cypress
# feat: Hide `projectId` section when project is not connected to dashboard

- closes https://github.com/cypress-io/cypress/issues/21806

### User-facing changelog
Fix an issue where there's a visible “Project ID" section (with no projectId) when user has not connected to dashboard.

### Additional details
The `projectId`  section should not show on the Dashboard if project has not been connected to dashboard.

### Steps to test
- Open a project not set up to record to the dashboard (no projectId)
- Open “Dashboard Settings”
- Verify that there is no Project Id displayed, only an action button to connect to dashboard

### How has the user experience changed?

BEFORE
When there's no projectId:
<img width="1390" alt="Screenshot 2022-09-14 at 6 45 44 AM" src="https://user-images.githubusercontent.com/45232708/190069700-eefe253c-1d71-4aea-a2e4-11f0a0a8e032.png">


AFTER
When there's no projectId:
<img width="1390" alt="Screenshot 2022-09-14 at 6 46 42 AM" src="https://user-images.githubusercontent.com/45232708/190069691-6a3b5092-2bb8-4d1b-8da1-1692abbdc0e6.png">



### PR Tasks
